raftstore-v2: exists compilation errors

## Bug Report

<!-- Thanks for your bug report! Don't worry if you can't fill out all the sections. -->

### What version of TiKV are you using?
<!-- You can run `tikv-server --version` -->
master

### What operating system and CPU are you using?
<!-- If you're using Linux, you can run `cat /proc/cpuinfo` -->
Linux kernel version: 
> Linux version 3.10.0-957.el7.x86_64 (mockbuild@kbuilder.bsys.centos.org) (gcc version 4.8.5 20150623 (Red Hat 4.8.5-36) (GCC) ) #1 SMP Thu Nov 8 23:39:32 UTC 2018

CPU info:
> Intel(R) Xeon(R) CPU E5-2630 v4 @ 2.20GHz

### Steps to reproduce
<!-- If possible, provide a recipe for reproducing the error. A complete runnable program is good. -->
Failed to compile the whole repo on master branch.

### What did you expect?
Compile successfully.

### What did happened?
```Cmake
Compiling raftstore-v2 v0.1.0 (/nvme1n1/lucasliang/workspace/tikv/components/raftstore-v2)

error[E0412]: cannot find type `FlushChannel` in this scope
   --> components/raftstore-v2/src/router/response_channel.rs:606:16
    |
239 | pub struct BaseChannel<Res> {
    | --------------------------- similarly named struct `BaseChannel` defined here
...
606 | impl Debug for FlushChannel {
    |                ^^^^^^^^^^^^ help: a struct with a similar name exists: `BaseChannel`
```

--- a/components/raftstore-v2/src/router/response_channel.rs
+++ b/components/raftstore-v2/src/router/response_channel.rs
@@ -603,6 +603,7 @@ pub type FlushChannel = BaseChannel<()>;
 #[cfg(feature = "testexport")]
 pub type FlushSubscriber = BaseSubscriber<()>;
 
+#[cfg(feature = "testexport")]
 impl Debug for FlushChannel {
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
         write!(f, "FlushChannel")

